### PR TITLE
Import corteza-webapp-compose as a package and extend corteza-base theme

### PR DESCRIPTION
**Pull request by darh**

### CODE REVIEW

File: src/themes/crust-base/index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import './index.scss'
+import 'corteza-webapp-compose/src/themes/corteza-base'
```
Comment (tjerman) [inline src/themes/crust-base/index.js:2]: Shouldn't this import order be turned around?

File: src/themes/crust-base/index.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+@import "variables.scss";
+@import "~corteza-webapp-compose/src/themes/corteza-base/index";
```
Comment (tjerman) [inline src/themes/crust-base/index.scss:2]: As well as here

File: package.json [hunk 3]
```diff
@@ -29,66 +29,69 @@
...
     "@panter/vue-i18next": "^0.15.0",
     "axios": "^0.18.0",
     "bootstrap-vue": "^2.0.0-rc.20",
     "c3": "^0.6.11",
     "chart.js": "^2.7.3",
-    "crust-api-client.js": "crusttech/crust-api-client.js.git#master",
-    "crust-common.vue": "crusttech/crust-common.vue.git#master",
-    "ejs": "^2.6.1",
+    "corteza-webapp-common": "cortezaproject/corteza-webapp-common.git#master",
+    "corteza-webapp-compose": "cortezaproject/corteza-webapp-compose.git#importable",
```
Comment (tjerman) [inline package.json:42]: Don't forget to change to `#master` :)

File: src/themes/crust-base/index.js (hunk 1 above)
Comment (darh) [reply src/themes/crust-base/index.js:2]: nope. SCSS variable value overriding does not work that way 🙄 
there is also a possibility that I've misinterpreted the docs 🤓

**Comment by tjerman** [comment]
You didn't delete unneeded tests/unit/*.spec files. Intentional?

File: src/themes/crust-base/index.js (hunk 1 above)
Comment (tjerman) [reply src/themes/crust-base/index.js:2]: Huh, I did a quick check and seems you are correct. Interesting o.o

**Comment by darh** [comment]
Not intentional. Removing now.

**Pull request closed without merging**